### ISSUE TITLE
[Visual Requirements – Make Code – Create a project] Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3632]
User Experience:
If luminosity ratio for text present on  tab will be less than the required threshold than users with low vision will not be able to see that text properly and they will miss important information.

Note: User credentials should NOT be included in the bug.

Repro Steps:

1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Basic, Input, Music, Led, more, Radio" controls and select it.
6. Navigate through all the controls appearing after selection of element.
7. Verify the issue.

Actual Result:
Luminosity ratio of  text present under  tab is less than 4.5:1.

Expected Result:
Luminosity ratio of  text present under  tab should be more than or equal to 4.5:1.

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://dequeuniversity.com/rules/axe/4.0/color-contrast?application=msftAI)

Target application: Microsoft MakeCode for micro:bit - https://makecode.microbit.org/#editor

Element path: div[aria-label="Toggle\ category\ Basic"] > .blocklyTreeLabel

Snippet: <span class="blocklyTreeLabel">Basic</span>

How to fix:
Fix any of the following:
  Element has insufficient color contrast of 3.23 (foreground color: #1e90ff, background color: #ffffff, font size: 13.8pt (18.4px), font weight: normal). Expected contrast ratio of 4.5:1

Note:
Same issue exists on Firefox & Chrome.

WCAG Reference:
[WCAG 1.4.3 – Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.4.3, #Win10-Edge(Chromium)

![Uploading WCAG1.4.3_K4w.png…]()
